icu4j/main/tests/core/src/com/ibm/icu/dev/test/calendar/JapaneseTest.java

```diff
@@ -234,10 +234,10 @@ public void Test3860()
 
     @Test
     public void TestForceGannenNumbering() {
-        final String jCalShortPattern = "y/M/d"; // Note: just 'y' doesn't work here.
-        final String jCalGannenDate = "1/5/9"; // A date in the above format after the accession date for Heisei [1989-] era (Heisei year 1 Jan 8)
-                                               // or Reiwa [2019-] era (Reiwa year 1 May 1). If before the accession date,
-                                               // the year will be in the previous era.
+//        final String jCalShortPattern = "y/M/d"; // Note: just 'y' doesn't work here.
```

Comment:
These are unused.. Obviously, we had some tests using them.
If it's permanent, we'd just delete them.